prusti-utils/src/config.rs

```diff
@@ -364,6 +368,11 @@ pub fn check_panics() -> bool {
     read_setting("check_panics")
 }
 
+/// When enabled, the time reasoning constrains are verified. Otherwise they are ignored.
```

Comment:
Please also update `flags.md` accordingly (for the [flags](https://viperproject.github.io/prusti-dev/dev-guide/config/flags.html) page in the dev guide).